org.civicrm.reqangular/src/common/templates/tab-outer.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <!--Overrides the 'uib/template/tabs/tabset.html'
+to add '{{tabset.customHeaderClass}}' which is used for custom header class-->
+  <ul class="nav nav-{{tabset.type || 'tabs'}} {{tabset.customHeaderClass}}"
+      ng-class="{'nav-stacked': vertical, 'nav-justified': justified}"
+      ng-transclude></ul>
+  <div class="tab-content">
+    <div class="tab-pane"
+         ng-repeat="tab in tabset.tabs"
+         ng-class="{active: tabset.active === tab.index}"
+         uib-tab-content-transclude="tab">
+    </div>
+  </div>
+</div>
```

Comment:
> Actually I think there is a better way to do this, without having to re-write the entire template just to add single string.

If you look here you can see that you can simply fetch the template, modify it, and then put it back in the template cache.

I think this is more maintainable because we keep using the default template instead of providing our own

I think having separate HTML is much more clean than fetching an HTML and modifying it with regex which makes it more complicated. Lets keep the HTML separated from JS